apps/web/components/PayPal/ApplePayButton.vue

```diff
@@ -19,105 +20,90 @@ const currency = computed(() => cartGetters.getCurrency(cart.value) || (useAppCo
 const applePayConfig = ref<ConfigResponse | null>(null);
 const paypal = await loadScript(currency.value);
 const localePath = useLocalePath();
-let isApplepayLoaded = false;
-
+const emits = defineEmits<{
+  (event: 'button-clicked', callback: PayPalAddToCartCallback): Promise<void>;
+}>();
 const loadApplePay = async () => {
   const scriptElement = document.createElement('script');
   scriptElement.setAttribute('src', 'https://applepay.cdn-apple.com/jsapi/v1/apple-pay-sdk.js');
   scriptElement.setAttribute('type', 'text/javascript');
   document.head.append(scriptElement);
 };
-const emits = defineEmits<{
-  (event: 'button-clicked', callback: PayPalAddToCartCallback): Promise<void>;
-}>();
+
+const setPaymentRequest = () => {
+  return {
+    countryCode: applePayConfig.value?.countryCode,
+    merchantCapabilities: applePayConfig.value?.merchantCapabilities,
+    supportedNetworks: applePayConfig.value?.supportedNetworks,
+    currencyCode: currency.value,
+    requiredShippingContactFields: [],
+    requiredBillingContactFields: ['postalAddress'],
```

Comment:
I think we wanted to remove that and make it an empty array

Reply:
this is not possible because we are using it here await applePay.confirmOrder({
            orderId: transaction.id,
            token: event.payment.token,
            billingContact: event.payment.billingContact,
          }); on the billingContact

Reply:
resolved